primitives/src/match_format.rs

```diff
@@ -53,6 +55,17 @@ pub fn trim_whitespace(input_string: StrLike) -> StrLike {
     result
 }
 
+pub fn match_side_effect(kind: &StrLike) -> Result<Bytes, &'static str> {
```

Comment:
align and cover all cases from `t3rn_protocol::side_effects::standards::standard_side_effects()`